src/Access/ReplicatedAccessStorage.h

```diff
@@ -82,8 +83,8 @@ class ReplicatedAccessStorage : public IAccessStorage
     AccessEntityPtr readImpl(const UUID & id, bool throw_if_not_exists) const override;
 
     mutable std::mutex mutex;
-    std::unordered_map<UUID, Entry> entries_by_id;
-    std::unordered_map<String, Entry *> entries_by_name_and_type[static_cast<size_t>(AccessEntityType::MAX)];
-    AccessChangesNotifier & changes_notifier;
+    std::unordered_map<UUID, Entry> entries_by_id TSA_GUARDED_BY(mutex);
+    std::unordered_map<String, Entry *> entries_by_name_and_type[static_cast<size_t>(AccessEntityType::MAX)] TSA_GUARDED_BY(mutex);
+    AccessChangesNotifier & changes_notifier; // XXX protected by mutex?
```

Comment:
Same

Reply:
Yepp, similar to the other place --> removed comment.